### ISSUE TITLE
How to set MSRP content-type?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#141]
```
What steps will reproduce the problem?
1.I want to set Content-type:text/plain;chatset=UTF-8 in MSRP message.
2.Content-type is set to Content-Type: application/octet-stream automatically.
3.How to set Content-type:text/plain;chatset=UTF-8 in MSRP message?

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
IMSDroid 1.1.343  on Android 2.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `nieyaol...@gmail.com` on 4 Jan 2011 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
ActionConfig messageConfig = new ActionConfig();
messageConfig.setMediaString(twrap_media_type_t.twrap_media_msrp, 
"content-type", "text/plain;charset=utf-8");
msrpSession.session.sendMessage(payload, (long)payload.capacity(), 
messageConfig);

setMediaString method do not work.

```

Original comment by `twbc.cla...@gmail.com` on 7 Jan 2011 at 6:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 7 Jan 2011 at 3:09
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
No comment？
```

Original comment by `twbc.cla...@gmail.com` on 7 Jan 2011 at 3:59

**Comment by GoogleCodeExporter** [comment]
```
There is no comment because I have to test it for confirmation. If confirmed 
then I will change the status to "NextRelease" ;)
```

Original comment by `boss...@yahoo.fr` on 7 Jan 2011 at 4:06

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.2.355
```

Original comment by `boss...@yahoo.fr` on 11 Jan 2011 at 1:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 11 Jan 2011 at 1:18
- Changed state: **Fixed**
